server/proto.go

```diff
@@ -28,6 +28,11 @@ func writeMessage(conn net.Conn, msgs []*message.Signed) error {
 		return err
 	}
 
+	_, err = conn.Write(frameMessage(reply))
+	if err != nil {
+		return err
+	}
+
```

Comment:
git interpreted this as moving the `Write` but the point is that `[Sent]` debug log needs to come after the actual write or else it could report a false success.